### DISCUSSION TITLE
Difficulty Installing R Packages

### DISCUSSION THREAD

**Discussion by abarenblitt** [MAAP-Project/Community#811] [Platform]
I'm having difficulty installing several R packages ("optmatch" specifically but I'm sure it's not the only one). Some I've been able to install using "mamba install -c conda-forge r-________" but this one I can't seem to install even directly through R. Has anyone figured out how to install packages like this through the MAAP?

**Comment by abarenblitt** [comment]
Updated to add that this particular package has been removed by CRAN, so this is a package specific issue it appears.

**Reply by wildintellect** [reply]
if `optmatch` isn't on CRAN anymore you'll need to do a direct from github install with devtools. This may not work if it's no longer compatible with recent versions of R.

**Reply by wildintellect** [reply]
Ok optmatch requires installing from source, which means we need some additional tools installed.

**Reply by wildintellect** [reply]
follow https://github.com/MAAP-Project/Community/issues/813 as we look for the answers which may also apply to RItools.

**Reply by wildintellect** [reply]
@abarenblitt I was able to RItools installed and loading see https://github.com/MAAP-Project/Community/issues/813#issuecomment-1741547746
